frontend/TestRun/ResourcesInfo.svelte

```diff
@@ -71,8 +78,27 @@
         });
     };
 </script>
-
-
+<div class="row">
+    <div class="p-1 m-2 d-flex align-items-center">
+        <button
+            type="button"
+            class="btn btn-outline-success me-2"
+            on:click={() => {
+                navigator.clipboard.writeText(
+                    "hydra clean-resources --backend " + backend + " --test-id " + test_id
+                );
+            }}>
+            <Fa icon={faCopy} /> Hydra Clean Resources
+        </button>
+        <a
+            href="https://jenkins.scylladb.com/view/QA/job/QA-tools/job/hydra-clean-test-resources/parambuild/?test_id={test_id}&backend={backend}"
```

Comment:
Will this open the `Build With Parameters` page or will this instantly start the build?

Reply:
will open with parameters filled